### ISSUE TITLE
FDroid 

### ISSUE THREAD

**Issue by PeterKW** [AbdurazaaqMohammed/AntiSplit-M#103]
Is this on the f droid store?

**Comment by AbdurazaaqMohammed** [comment]
No and I didn't want to add it for a few reasons mainly

- My code is terrible
- No one releases FOSS apps as split APKs. Practically this app can only be used for merging closed source apks, I don't think it will make a lot of sense for F-Droid

**Comment by PeterKW** [comment]
Does it send a notification if there's an updated version to download from GitHub?

**Comment by PeterKW** [comment]
Does it send a notification if there's an updated version to download from GitHub?

**Comment by PeterKW** [comment]
Sorry don't know how that sent twice

**Comment by drogga** [comment]
@PeterKW No, it doesn't, it only prompts you to DL it when you launch it, assuming that there's a new release and not a replaced .apk as a quick fix and if the option to check for updates is enabled in the settings (it is by default).

<details>You can delete the double comment and the one after it ;)</details>

**Comment by Rikj000** [comment]
> No and I didn't want to add it for a few reasons mainly

@AbdurazaaqMohammed I hope you could please re-consider that.   
It would offer end users an easy way to remain up-to-date with the latest and official versions of your app,   
through the most popular and commonly used FOSS app store available,   
without needing to resort to things like [Obtainium](https://obtainium.imranr.dev/) to stay up-to-date.

>     * My code is terrible

I haven't reviewed your code, but even if that's the case, it doesn't matter for the end users.   
Even if the underlying code is ugly, the app still works well.

>     * No one releases FOSS apps as split APKs.
>     Practically this app can only be used for merging closed source apks,
>     I don't think it will make a lot of sense for F-Droid.

F-Droid is for open source apps only, yours being one of them,   
so it makes sense to add yours to it.   
Even if your app is mostly used to merge closed source apks,   
your app still is open source.

Hope you'll re-consider, either way, thank you for AntiSplit-M! 😊

**Comment by solonovamax** [comment]
I like fdroid because not only does it offer an easy way to stay up to date, but it also means I can know that the apk I'm installing is built from the code that is on github, whereas that's not necessarily the case with github releases.
